py/circuitpy_mpconfig.mk

```diff
@@ -90,6 +90,9 @@ CFLAGS += -DCIRCUITPY_BLEIO=$(CIRCUITPY_BLEIO)
 CIRCUITPY_BOARD ?= 1
 CFLAGS += -DCIRCUITPY_BOARD=$(CIRCUITPY_BOARD)
 
+CIRCUITPY_BUSDEVICE ?= 1
```

Comment:
This may need to be `CIRCUITPY_BUSDEVICE ?= $(CIRCUITPY_FULL_BUILD)`

Reply:
The small M0 builds are not building with bus device included. I can mark them to exclude it like is done for other libraries?